clang/include/clang/Sema/Sema.h

```diff
@@ -13164,13 +13175,6 @@ void Sema::AddOneConstantValueAttr(Decl *D, const AttributeCommonInfo &CI,
       return;
     E = ICE.get();
   }
-
-  if (IntelFPGAPrivateCopiesAttr::classof(&TmpAttr)) {
```

Comment:
We do not use AddOneConstantValueAttr() any more. I think we can remove this redundant function.

Reply:
It is still used when handling `IntelFPGAForcePow2DepthAttr`. I can rename it to `addIntelFPGAForcePow2Depth()` if you'd like. I wasn't sure if other FPGA attributes (For example, other attributes with ranges) could/would be refactored to use this function in the future, and so I left it as it is. Should I rename it?

Reply:
Thanks for the clarification. I did not realize that It is still used when handling IntelFPGAForcePow2DepthAttr. Rename is not needed here.